src/guides/v2.4/comp-mgr/trouble/current-search-engine.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Should we add a section about the case when you get an error about connecting to ES.
`Could not validate a connection to Elasticsearch. No alive nodes found in your cluster`
fyi, in this error message "Could not validate a connection to Elasticsearch" is consistent, but the second part is from ES and may vary depending on the issue (not sure of all possibilities)

Reply:
ok, sounds good to me 👍